sorc/build_all.sh

```diff
@@ -82,19 +64,95 @@ while getopts ":a:dfghj:kuvwy" option; do
       ;;
   esac
 done
-
 shift $((OPTIND-1))
 
+supported_systems=("gfs" "gefs" "sfs" "gsi" "gdas" "all")
+# shellcheck disable=SC2034
+gfs_builds="gfs gfs_utils ufs_utils upp ww3_gfs"
+# shellcheck disable=SC2034
+gefs_builds="gefs gfs_utils ufs_utils upp ww3_gefs"
+# shellcheck disable=SC2034
+sfs_builds="sfs gfs_utils ufs_utils upp ww3_gefs"
+# shellcheck disable=SC2034
+gsi_builds="gsi_enkf gsi_monitor gsi_utils"
+# shellcheck disable=SC2034
+gdas_builds="gdas gsi_monitor gsi_utils"
+# shellcheck disable=SC2034
+all_builds="gfs gfs_utils ufs_utils upp ww3_gfs ww3_gefs gdas gsi_enkf gsi_monitor gsi_monitor gsi_utils"
```

Comment:
I think this is missing `gefs` and `gsi_monitor` is duplicated
```suggestion
all_builds="gfs gfs_utils ufs_utils upp ww3_gfs gefs ww3_gefs gdas gsi_enkf gsi_monitor gsi_utils"
```